jib-core/src/main/java/com/google/cloud/tools/jib/api/LayerConfiguration.java

```diff
@@ -289,6 +304,10 @@ public String getName() {
    * @return the list of layer entries
    */
   public ImmutableList<LayerEntry> getLayerEntries() {
-    return layerEntries;
+    return ImmutableList.copyOf(entries);
```

Comment:
This is still guava no?

Reply:
True. We'll keep the jib-core API unchanged. When I duplicate this into jib-build-plan, the method name will change to `getEntries()` and it will return the plain `List`.